<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="positionGroupBox">
      <property name="title">
       <string>Position &amp;&amp; Fix</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Longitude:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Latitude:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="eastWestComboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>E</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>W</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="northSouthComboBox">
           <item>
            <property name="text">
             <string>N</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>S</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="latitudeSpinBox">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="longitudeSpinBox">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Altitude:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="altitudeSpinBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>68</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>km</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Fix:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="gpsFixComboBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>No GPS Fix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2D Fix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3D Fix</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="movementGroupBox">
      <property name="title">
       <string>Movment</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="bearingSpinBox">
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="singleStep">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>km/h</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSpinBox" name="speedSpinBox">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bearing:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>22</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>deg</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>318</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="xksimulator.qrc">
     <normaloff>:/images/icons/close-16.png</normaloff>:/images/icons/close-16.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="xksimulator.qrc">
     <normaloff>:/images/icons/refresh-16.png</normaloff>:/images/icons/refresh-16.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Devices</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="xksimulator.qrc">
     <normaloff>:/images/icons/about-16.png</normaloff>:/images/icons/about-16.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="xksimulator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
